### DISCUSSION TITLE
I would like to translate the docu to german

### DISCUSSION THREAD

**Discussion by DenisMtfl** [dotnetcore/BootstrapBlazor#3610] [Q&A]
### Which class is this unit test associated with?

Can you publish the website-code to add translation as pull request?

**Comment by bb-auto** [comment]
@DenisMtfl Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@DenisMtfl thanks. you can find them as follow 
https://github.com/dotnetcore/BootstrapBlazor/blob/main/localization/de.json

document localization folder

https://github.com/dotnetcore/BootstrapBlazor/tree/main/src/BootstrapBlazor.Server/Locales
